lambdas/data-migration2/src/storeErrors.ts

```diff
@@ -0,0 +1,25 @@
+import { s3PutObject } from '@cumulus/aws-client/S3';
+
+/**
+ * Store migration errors in JSON file in S3
+ * @param {string} bucket
+ * @param {string[]} message
+ * @param {string} recordClassification
```

Comment:
Definitely open to better names because I am not sure this is it

Reply:
I spent longer than I should trying to improve on that, and came up empty.   This seems fine, particularly in this case as it really doesn't present a readability concern.

Reply:
I would ask for descriptions of each param in the JSdoc though

Reply:
Also, `migrationName`?